koku/masu/database/trino_sql/aws/openshift/reporting_ocpawscostlineitem_project_daily_summary_p.sql

```diff
@@ -42,6 +42,12 @@ INSERT INTO postgres.{{schema | sqlsafe}}.reporting_ocpawscostlineitem_project_d
     cost_category_id,
     source_uuid
 )
+with cte_pg_enabled_keys as (
+    select array_agg(key order by key) as keys
```

Comment:
We need to include the specific kubevirt key here for back populate to work correctly.